securityconfig/roles.yml

```diff
@@ -38,3 +38,25 @@ alerting_full_access:
         - 'indices_monitor'
         - 'indices:admin/aliases/get'
         - 'indices:admin/mappings/get'
+
+# Allow users to read Anomaly Detection detectors and results
+ad_read_access:
```

Comment:
`ad` -> `anomaly`

Reply:
yeah we internally use ad as anomaly detection, do you recommend expanding it?

Reply:
yes, 'ad' is too short and may mean not only anomaly detection.

Reply:
Sure will rename it do anomaly.